packages/paste-website/src/components/tokens-list/index.tsx

```diff
@@ -47,6 +49,32 @@ interface TokensListProps {
 }
 
 const getTokensByTheme = (theme: ThemeVariants, props: TokensListProps): TokenCategory[] => {
+  const font = props.consoleTokens[0].node.tokens.find(ele => ele.categoryName === 'fonts');
+  if (font) {
+    font.info = (
+      <Callout variant="secondary">
+        <CalloutTitle as="h4">Heads up about fonts in Paste!</CalloutTitle>
+        <CalloutText>
+          Due to font-family licensing, we have no ability to redistribute certain themed fonts. Paste makes the
```

Comment:
Paste assumes that you have the correct fonts are loaded for your product (Whitney for Console-themed UIs, Colfax for SendGrid-themed UIs)